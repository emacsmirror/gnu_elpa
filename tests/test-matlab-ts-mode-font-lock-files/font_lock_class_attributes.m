% -*- matlab-ts -*-
% See https://github.com/acristoffers/tree-sitter-matlab/issues/29
classdef font_lock_class_attributes
   methods (Access = protected)
      function out = method1(obj,inputArg)
         ...
      end
   end
   methods (Access = private)
      function out = method2(obj,inputArg)
          ...
      end
   end
end
