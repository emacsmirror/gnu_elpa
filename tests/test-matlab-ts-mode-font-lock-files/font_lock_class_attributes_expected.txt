c ccc ccccccccc ccc
c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
kkkkkkkk ffffffffffffffffffffffffff
   kkkkkkk btttttt o BBBBBBBBBb
      kkkkkkkk vvv o fffffffbvvvDvvvvvvvvb
         ccc
      kkk
   kkk
   kkkkkkk btttttt o BBBBBBBb
      kkkkkkkk vvv o fffffffbvvvDvvvvvvvvb
          ccc
      kkk
   kkk
kkk
